docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -175,35 +163,43 @@ This should not even be possible, but it means that an unhandled error was raise
 > Event Prefix: `Summarizer:Running:`
 
 - `summarizeCount` - the number of summarize attempts this client has made. This can be used to correlate events for individual summary attempts.
+- `summarizerSuccessfulAttempts` - the number of successful summaries this summarizer instance has performed
 
 ### SummaryAckWaitTimeout
 
 > Error
 
 When a summary op is sent, the summarizer waits `summaryAckWaitTimeout` for a summary ack/nack op in response from the server. If a corresponding response is not seen within that time, this event is raised, and the client retries.
 
-- `refSequenceNumber` - last attempt summary op reference sequence number.
+- `maxAckWaitTime` - cap on the maximum amount of time client will wait for a summarize op ack
+- `referenceSequenceNumber` - last attempt summary op reference sequence number.
 - `summarySequenceNumber` - last attempt summary op sequence number.
 - `timePending` - time spent waiting for a summary ack/nack as computed by client.
 
 ### MissingSummaryAckFoundByOps
 
 During first load, the wait for a summary ack/nack op in response to a summary op, can be bypassed by comparing the op timestamps. Normally a timer is used while running, but if the server-stamped op time difference exceeds the `maxAckWaitTimeout`, then raise this event, clear the timer and stop waiting to start.
 
-- `refSequenceNumber` - last attempt summary op reference sequence number.
+- `referenceSequenceNumber` - last attempt summary op reference sequence number.
 - `summarySequenceNumber` - last attempt summary op sequence number.
 
 ### SummarizeAttemptDelay
 
-- `retryAfterSeconds` - delay from `retryAfter` found in the summary nack response op. This will override any regular delay time.
-- `regularDelaySeconds` - delay from regular summarize attempt retry.
+Logs the presence of a delay before attempting summary
+
+- `duration` - duration delay in seconds. This is the `retryAfter` value found in the summary nack response op, if present.
+Otherwise, it's the delay from regular summarize attempt retry.
+- `reason` - "nack with retryAfter" if the `duration` value came from a summary nack response op. Undefined otherwise.
 
 ### FailToSummarize
 
 > Error
 
 All consecutive retry attempts to summarize by heuristics have failed. The summarizer client should stop itself with "failToSummarize" reason code, closing the container.
 
+- `summarizeReason` - reason for attempting to summarize
+- `message` - message returned with the last summarize result
```

Comment:
unless `reason` is already used on this event, I'd prefer to use `reason` over `message`.
Mostly because `reason` and `error` are properties that are the most used, and usually are present on all "exploration" queries, so it's easier to get overview of the session without need to look at the code / this readme.md

Reply:
Yeah, reason is already present. I'm just updating the documentation to match what we log. Are you saying you'd like to stop logging "message"?